llvm/test/MC/ELF/osabi-freebsd.s

```diff
@@ -0,0 +1,2 @@
+# RUN: llvm-mc -filetype obj -triple amd64-freebsd %s | llvm-readobj -hS - | FileCheck %s
```

Comment:
I added `osabi.s` to unify FreeBSD/Linux/Solaris testing. Consider adding OpenBSD tests there instead of adding a new file.